src/routes/blog/post/cors-error/+page.markdoc

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: Solving CORS errors with Appwrite
+description: Understanding why you are getting CORS error when sending a request to an appwrite backend and how to debug.
+date: 2024-01-16
+cover: /images/blog/cors-error/cors_cover.png
+timeToRead: 3
+author: dennis-ivy
+category: engineering
+featured: false
+---
+
+I want to address an issue I've seen popping up on stack overflow and the appwrite discord server and address some of the reasons you may be getting this error, and walk you through some of the steps you can take to try and resolve it as well.
+
+The error message you'll see in your console when trying to make a request to an Appwrite backend will look something like this: `Access blocked by CORS policy.`
+
+## **Understanding CORS**
```

Comment:
We start with `#` in the markdown -> It will be transformed to `##` Please un bold the  heading.